src/poly/poly.jl

```diff
@@ -33,12 +33,19 @@ nvars(R::PolyRing) = Int(libSingular.rVar(R.ptr))
 
 @doc Markdown.doc"""
     has_global_ordering(R::PolyRing)
-> Return `true` if the given polynomial has a global ordering, i.e. if $1 < x$ for
+> Return `true` if the given ring has a global ordering, i.e. if $1 < x$ for
 > each variable $x$ in the ring. This include `:lex`, `:deglex` and `:degrevlex`
-> orderings..
+> orderings.
 """
 has_global_ordering(R::PolyRing) = Bool(libSingular.rHasGlobalOrdering(R.ptr))
 
+@doc Markdown.doc"""
```

Comment:
However, underscore are required for lower case names. The one exception is after the word "is". I know, crazy convention, but that is our convention. So this should be "isquotient_ring"

Reply:
Done.